great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -139,6 +139,7 @@
             BigQueryTypes = namedtuple("BigQueryTypes", sorted(sqla_bigquery._type_map))
             bigquery_types_tuple = BigQueryTypes(**sqla_bigquery._type_map)
     except (ImportError, AttributeError):
+        sqla_bigquery = None
```

Comment:
Creates a possible null ref on line 1711

Reply:
and line 2113

Reply:
likewise ^